pkg/catalog/mock_catalog_generated.go

```diff
@@ -108,7 +108,6 @@ func (m *MockMeshCataloger) GetResolvableHostnamesForUpstreamService(arg0, arg1
 	return ret0, ret1
 }
 
-// TODO : remove as a part of routes refactor (#2397)
```

Comment:
nit: lets not remove these comments in this PR please

Reply:
These are autogenerated mocks, we should anyway have to regenerate them when we update the interface.